test/unit/index.test.js

```diff
@@ -68,6 +69,20 @@ describe('convert', function () {
     });
 
   });
+
+  it('Should return Not WSDL Specification found in your document whit invalid file', function () {
```

Comment:
This error should be `Provided document is not a valid WSDL specification`. As current error is confusing and doesn't convey correct information to user.

Also correct typo `whit` to `with`

Reply:
Added in last commits